src/components/BillSplit/BillSplit.js

```diff
@@ -0,0 +1,46 @@
+import React, { useState } from 'react';
+import '../TipCalc/TipCalc.css';
+
+export default function BillSplit() {
+
```

Comment:
Way to use hooks! And I like how your naming conventions for your functions and variables